src/main/java/attendance/domain/Attendance.java

```diff
@@ -0,0 +1,79 @@
+package attendance.domain;
+
+import java.time.LocalDate;
+import java.time.LocalDateTime;
+import java.time.LocalTime;
+import java.util.Objects;
+
+public class Attendance {
+
+    private final String nickname;
+    private final AttendanceDate attendanceDate;
+    private LocalTime attendanceTime;
+
+    public Attendance(String nickname, LocalDate attendanceDate, LocalTime attendanceTime) {
+        this(nickname, LocalDateTime.of(attendanceDate, attendanceTime));
+    }
+
+    public Attendance(String nickname, LocalDateTime attendanceDateTime) {
+        this.nickname = nickname;
+        this.attendanceDate = new AttendanceDate(attendanceDateTime.toLocalDate());
+        this.attendanceTime = attendanceDateTime.toLocalTime();
+    }
+
+    public boolean isAlreadyAttend(Attendance newAttendance) {
+        return nickname.equals(newAttendance.nickname) &&
+                attendanceDate.equals(newAttendance.attendanceDate);
+    }
```

Comment:
메서드 오버로딩을 활용할 때는 각각 구현하기보다는 메서드 체이닝을 활용해보시면 좋을 것 같아요. 😃 
```suggestion
    public boolean isAlreadyAttend(Attendance newAttendance) {
        return isAlreadyAttend(newAttendance.nickname, newAttendance.attendanceDate);
    }
```

Reply:
감사합니다! 확실히 오버로딩시 메서드 체이닝을 활용하는게 좋아보이네요:)

Reply:
[refactor: Attendance isAlreadyAttend method 메서드 체이닝 활용하여 리팩터링](https://github.com/woowacourse/java-attendance/pull/126/commits/a729aa6e271eb31150f224aa319165f09f026bd6)